documentation/pom.xml

```diff
@@ -181,6 +181,9 @@
     <profiles>
         <profile>
             <id>documentation-pdf</id>
+            <activation>
+                <activeByDefault>!skipPdfDocumentation</activeByDefault>
```

Comment:
This is not a good idea. `activeByDefault` means "active if no other profile is enabled" (don't ask me why, just another maven weirdness).
In our case, we have other profiles, so this change didn't make the documentation-pdf profile activate by default.

You may want to use this instead:

```
            <activation>
                <property>
                    <name>!skipPdfDocumentation</name>
                </property>
            </activation>
```